Invoro.WebApp/Invoro/invoro/src/lib/components/Vote/Vote.component.tsx

```diff
@@ -21,7 +21,7 @@ class VoteComponent extends React.Component<VoteProps> {
     render() {
         return (
             <IconButton color="primary" onClick={() => this.handleOnClick()} aria-label="Vote for the feature">
-                {this.props.isVoted ? <StarIcon /> : <StarBorderIcon />}
+                {this.props.isVoted ? <VoteIconComponent /> : <UnvoteIconComponent />}
```

Comment:
Not the opposite? In case  IsVoted=true, it means that it is already voted and you can Unvoteit, so you need to render the UnvoteIcon. No?

Reply:
the icon rendered according to the isVoted prop.
isVoted=true means the feature is voted, than the VoteIconComponent should be rendered
isVoted=false means the feature is not voted, than the UnvoteIconComponent should be rendered